src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
 Once executed, this query will return a single Workflow named _Create OpenStack Instance_.
 
+=== Send Workflows to another Catalog Bucket, from another Scheduler or the local one.
+
+In the Workflow Catalog, it is possible to select Workflows from the selected bucket and send them to another bucket. This bucket can belong to the same Catalog or to another Scheduler's Catalog.
+
+==== Send Workflows directly
+
+If the Workflows to share are sent to a Scheduler having a public API, or to another bucket of the local Scheduler, it is convenient to send them directly.
+
+Select the bucket then the Workflows. Choose `Send to another scheduler` and click `OK`.
+
+==== Export Workflows into an archive, then Import them
+
+If the remote Scheduler is running in another network, of if you cannot use direct sending for some reason, you can export Workflows into an archive, that you can transfer by any file transfer method (email, usb key, etc.) and then import this archive into the other Scheduler.
+
+To export the selected Workflows as a ZIP archive, select a bucket, then the targeted Workflows. Choose "Export selected Workflows" into the list and click on `OK`. You'll get a ZIP file that you'll be able to tranfer to the machine on which the other Scheduler runs.
```

Comment:
To be consistent with the previous instruction, I would start by:

First, select the bucket and the Workflows to export, then choose [...]

IMHO `that you'll be able to tranfer [...]` is confusing, I would simply put `that can be then imported on another Scheduler.` for example.